notebooks/250-efficient-object-detection/250-efficient-object-detection.ipynb

```diff
@@ -0,0 +1,622 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    model_ir = mo.convert_model(saved_model_dir=str(MODEL_DIR_PATH))
</pre><p>serialize(model_ir,str(XML_PATH)) - please fix code style (not enough white space)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>